code_annotations/config_and_tools/config/setting_annotations.yaml

```diff
@@ -0,0 +1,11 @@
+source_path: ./
+report_path: reports
+safelist_path: .annotation_safe_list.yml
+annotations:
+  setting:
+    - ".. setting_name:":
+    - ".. setting_default:":
+    - ".. setting_description:":
```

Comment:
Do you want a `setting_warning`?

Reply:
Yes, I didn't think about it but it would make sense.